projects/e2e/workbench/src/util/testing.util.ts

```diff
@@ -90,3 +90,15 @@ export async function hasBrowserError(error: string): Promise<boolean> {
   const logs = await this.browserErrors();
   return logs.some(log => log.message.includes(error));
 }
+
+/**
+ * Press key and perform click on given element.
+ *
+ * It is important to release the pressed key by {@link #keyUp} in order to avoid side effects in other tests.
+ *
+ * This problem is addressed at http://www.gjermundbjaanes.com/obscure-test-failure-after-using-keydown-with-protractor/
+ */
+export async function clickElement(elementFinder: ElementFinder, pressKey: string): Promise<void> {
```

Comment:
The same applies here. Please see the comment above.

Reply:
done